client/authorization/reducer.js

```diff
@@ -4,71 +4,98 @@ import * as actions from './actions'
 const initialState = {
     isShown: false,
     email: '',
-    password: ''
+    password: '',
+    username: ''
 };
 
 const reducer = handleActions(
     {
-        [actions.init]: (state) => ({
+        [actions.init]: state => ({
             ...state,
             isShown: initialState.isShown,
```

Comment:
why do you override something there? Shouldn't state already be in the initial state?